client/src/components/Forms/MarkdownInput.tsx

```diff
@@ -58,6 +58,13 @@ export default function MarkdownInput({ historicalCommentSelected }: Props) {
               <ReactMarkdown rehypePlugins={[remarkGfm]}>{text}</ReactMarkdown>
             </Typography.Text>
           </div>
+          {!text ? (
+            <Typography.Paragraph type="danger">Please leave a comment</Typography.Paragraph>
+          ) : text.length < 30 ? (
+            <Typography.Paragraph type="danger">Please leave a detailed comment</Typography.Paragraph>
+          ) : (
+            ''
```

Comment:
Returning null is usually the best idea if you intend to indicate that no data is available.